recipe/meta.yaml

```diff
@@ -23,14 +23,14 @@ requirements:
     - setuptools
   run:
     - python
-    - numpy >=1.21,<2  # [py>=310 or (osx and arm64)]
-    - numpy >=1.20,<2  # [py<310 and not (osx and arm64)]
+    - numpy >=1.22
     - opt_einsum
-    - scipy >=1.5
+    - scipy >=1.7
     - typing_extensions
```

Comment:
```suggestion
```
I'm not 100% confident on this one. But at least thw original wheel doesn't declare a dependency on typing-extensions. See https://inspector.pypi.io/project/jax/0.4.16/packages/2f/7e/052564673066fcb143a7d096232790d037554b323f0cf0e7c2af3a60ca88/jax-0.4.16-py3-none-any.whl/jax-0.4.16.dist-info/METADATA.